src/message.ts

```diff
@@ -604,6 +604,34 @@ export class Message implements Sayable {
     ])
   }
 
+  /**
+   * Return the recalled message id
+   *
+   * Valid only if {@link Message#type} is equal to `MsgType.RECALLED`
+   *
+   * @returns {string} msgId
+   * @memberof Message
+   * @example
+   * ```javascript
+   * Bot.on('message', async m => {
+   *   if (m.type() === MsgType.RECALLED) {
+   *     await m.say(`You recalled a message, message ID：${m.getRevokeId()}`)
+   *   }
+   * })
+   * ```
+   */
+  public getRevokeId(): string {
```

Comment:
I see.

As my understanding now, the parameter of `_recalledMsgProcess()` is `<sysmsg type="revokemsg">...</sysmsg>`, right?

If so, I think the better procedure might be(in wechaty-bro.js):

1. find the original message for that `msgid`, then 
1. set the message with MsgType.RECALLED, then 
1. use `WechatBro.emit()` to emit that message again. (see: https://github.com/Chatie/wechaty/blob/master/src/puppet-web/wechaty-bro.js#L366)

The above procedure could let the end-user receive a `message` event, with the original message, but with MsgType.RECALLED set. (and no new method need to be introduced)

How about this idea, what do you think?